gemrb/core/GUI/EventMgr.cpp

```diff
@@ -363,7 +363,8 @@ void EventMgr::KeyPress(unsigned char Key, unsigned short Mod)
 	Control *ctrl = last_win_focused->GetFocus();
 	if (!ctrl || !ctrl->OnKeyPress( Key, Mod )) {
 		// FIXME: need a better way to determine when to call ResolveKey/SetHotKey
-		if (core->GetGameControl()
+		if (!last_win_focused->OnKeyPress(Key, Mod)
+			&& core->GetGameControl()
```

Comment:
Does switching windows still work via hotkeys after this? Eg. inventory to guirec or journal.

I'm just about to leave for a week, so couldn't take but a cursory look.